code/fmri/reports.py

```diff
@@ -57,7 +59,7 @@
 FC_FIGURE_SIZE: tuple = (70, 45)
 LABELSIZE: int = 42
 NETWORK_CMAP: str = "turbo"
-N_PERMUTATION: int = 10000
+N_PERMUTATION: int = 10
```

Comment:
I'm guessing you edited this for debugging.

```suggestion
N_PERMUTATION: int = 10000
```

Reply:
ah yes good catch, indeed